ext/phar/util.c

```diff
@@ -1893,7 +1893,9 @@ int phar_create_signature(phar_archive_data *phar, php_stream *fp, char **signat
 
 			siglen = EVP_PKEY_size(key);
 			sigbuf = emalloc(siglen + 1);
-
+			if (md_ctx == NULL){
+				return FAILURE;
```

Comment:
should this be done a few lines above, as it looks like it leaks `sigbuf` at this line

Reply:
Sorry I didn't notice `sigbuf` before, I'll fix it